src/pure.js

```diff
@@ -11,45 +11,21 @@ const componentCache = new Set()
 
 const svelteComponentOptions = IS_SVELTE_5
   ? ['target', 'props', 'events', 'context', 'intro', 'recover']
-  : ['accessors', 'anchor', 'props', 'hydrate', 'intro', 'context']
-
-const render = (
-  Component,
-  { target, ...options } = {},
-  { container, queries } = {}
-) => {
-  container = container || document.body
-  target = target || container.appendChild(document.createElement('div'))
-  targetCache.add(target)
+  : ['target', 'accessors', 'anchor', 'props', 'hydrate', 'intro', 'context']
 
-  const ComponentConstructor = Component.default || Component
+const render = (Component, componentOptions = {}, renderOptions = {}) => {
+  componentOptions = checkProps(componentOptions)
 
-  const checkProps = (options) => {
-    const isProps = !Object.keys(options).some((option) =>
-      svelteComponentOptions.includes(option)
-    )
+  const baseElement =
+    renderOptions.baseElement ?? componentOptions.target ?? document.body
 
-    // Check if any props and Svelte options were accidentally mixed.
-    if (!isProps) {
-      const unrecognizedOptions = Object.keys(options).filter(
-        (option) => !svelteComponentOptions.includes(option)
-      )
-
-      if (unrecognizedOptions.length > 0) {
-        throw Error(`
-          Unknown options were found [${unrecognizedOptions}]. This might happen if you've mixed
-          passing in props with Svelte options into the render function. Valid Svelte options
-          are [${svelteComponentOptions}]. You can either change the prop names, or pass in your
-          props for that component via the \`props\` option.\n\n
-          Eg: const { /** Results **/ } = render(MyComponent, { props: { /** props here **/ } })\n\n
-        `)
-      }
+  const target =
+    componentOptions.target ??
+    baseElement.appendChild(document.createElement('div'))
 
-      return options
-    }
+  targetCache.add(target)
 
-    return { props: options }
-  }
+  const ComponentConstructor = Component.default || Component
```

Comment:
I think this must be a weird holdover from CJS days. I may remove it in a future PR, since I can't figure out how to actually trigger this code-path in tests now that we're ESM-native